README.md

```diff
@@ -1,9 +1,9 @@
 Rundeck WinRM Plugin
 --------------------
 
-This is a [Rundeck Node Execution plugin][1] that uses WinRM to connect to Windows and execute commands.  It uses the [OverThere Library][2] to provide the WinRM implementation, and uses Basic authentication over HTTPS.
+These are a [Rundeck Node Execution plugin and a Rundeck File Copier plugin][1] that uses WinRM to connect to Windows and execute commands.  It uses the [OverThere Library][2] to provide the WinRM implementation, and uses Basic authentication over HTTPS.
```

Comment:
Split this in two links, are there are two different pages:
- http://rundeck.org/docs/plugins-user-guide/node-execution-plugins.html
- http://rundeck.org/docs/developer/file-copier-plugin.html

Reply:
These are not the equivalents links. FileCopier and NodeExecutor are "node execution" plugin category
But agree to add the link to the considered plugin